docs: add installation instructions for Tiptap UI Components (#228)
ueberdosis/tiptap-docs: src/content/ui-components/getting-started/overview.mdx

---
title: Integrate Tiptap UI Components
meta:
  title: UI Components | Tiptap

>   description: Set up Tiptap UI Components using one of our prebuilt templates or by integrating individual components. More in the docs!

Reviewer: ```suggestion
  description: Set up Tiptap UI Components with a prebuilt template or by integrating individual components. More in the docs!
```

Simplified the sentence.

A few days we talked about avoiding using 1st person singular ("we") in the docs. To speak from the developer/customer's perspective.
